parlai/mturk/tasks/acute_eval/example_script.py

```diff
@@ -0,0 +1,54 @@
+from parlai.mturk.tasks.pairwise_dialogue_eval.run import main as run_main, add_args
+
+"""
+Example script for running ACUTE-EVAL.
+The only arguments that need to be modified for this to be run are:
+'dialogs_path':  Path to folder containing logs of all model conversations.
+    Each model should have one file named '<modelname>.jsonl'
+'model_comparisons': List of tuples indicating pairs of models to be compared.
+    The model names here should match the names of files in the folder
+'onboarding_tasks': List of tuples in format (id1, id2, name of matchup) where
+    id1, id2 are conversation ids where id2 is the correct conversation's id
+Help strings for the other arguments can be found in run.py
+"""
+
+
+def set_args():
+    args = add_args()
+    args['dialogs_path'] = '/home/ParlAI/parlai/mturk/tasks/acute_eval/example'
```

Comment:
can we set this from the person's parlai home variable? it might not be in /home/ParlAI

Reply:
this was only meant as an example - I expect people to copy this script and modify all of the args

Reply:
Could this be another command line param? (And put in the README the format it expects the file to be in?) It would be nice also to specify what it does if you give it a directory versus what it does when you get it a file and have the default behavior do the right. In addition, one nice thing would be if someone just has the ability to implement a single function that parses their file. (So pull this out into its own function that can be overridden:  https://github.com/facebookresearch/ParlAI/blob/8a3018dc8e2f4f5f6992c2a7058c27fb267b1524/parlai/mturk/tasks/acute_eval/run.py#L163 ) The frontend expects certain fields to be there such as "dialog",  so the format should be specified to.

Reply:
Could this be another command line param? (And put in the README the format it expects the file to be in?)

Reply:
It actually is a CLI arg, Margaret chose to set it via a script.